lib/cc/analyzer/container.rb

```diff
@@ -172,6 +172,7 @@ def kill_reader_threads
       def reap_running_container(message)
         Analyzer.logger.warn("killing container name=#{@name} message=#{message.inspect}")
         POSIX::Spawn::Child.new("docker", "kill", @name)
+        POSIX::Spawn::Child.new("docker", "wait", @name)
```

Comment:
In the past we've seen zombie engines that continued to run even after being killed. If that were to happen, this would result in builder effectively zombieing as well as it would wait for something that isn't actually going to die. That might make a zombie engine more visibile, which would be good, but it might also just make the problem worse. Thoughts on that?